.travis.yml

```diff
@@ -1,5 +1,33 @@
 language: python
 dist: xenial  # required for Python >= 3.7
 python: 3.7
-install: pip install flake8
-script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+cache: pip
+install: pip install black flake8 mypy pytest
+before_script:
+  - black --check . || true
+  - flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+  # - python -m doctest -v other/detecting_english_programmatically.py
+  - python -m doctest
+      ciphers/affine_cipher.py
```

Comment:
I think we should avoid doctests, and change the code accordingly, otherwise, .travis.yml will need to be updated when such code is written in new files. Although writing doctest is good, but since most of the files are not implemented that way, so let's maintain the uniformity.

Reply:
Pytest now runs the doctests.